src/main/app/src/utils/haku/getHakuFormConfig.ts

```diff
@@ -72,23 +72,30 @@ const config = createFormConfigBuilder().registerSections([
         field: '.hakuaikaGroup',
         required: true,
       },
+      {
+        field: '.ajankohtaKaytossa',
+      },
       {
         field: '.ajankohtaTyyppi',
-        validate: validateIfJulkaistu(
-          validateExistence('aikataulut.ajankohtaTyyppi')
-        ),
+        validate: eb =>
+          validateIf(
+            eb?.values?.aikataulut?.ajankohtaKaytossa &&
```

Comment:
Mä läiskäsin eb:lle privateks valuesit, ne saa kyllä sieltä kutsumalla eb.getValues()
Tonhan voi toki kirjottaa sinne classigetterinäkin että tää eb.values syntaksi pätee yhä:

get values() { return this.values; }

Reply:
Taidanpa tehdä tuon getterin. Ehkä hyväkin tehdä noin, kun eikös meillä ollut puhetta, että jatkossa validointifunktioille välitettävä arvo olisi mahdollisesti pelkkä objekti?

Reply:
Joo oishan se kiva ettei täs tarvis luokilla puljata, mut se ois nice-to-have. Toimii tää tälleeki